artiq/applets/arg_modifier.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+from PyQt5 import QtWidgets
+
+from artiq.applets.simple import SimpleApplet
+
+
+class ArgumentModifier(QtWidgets.QLabel):
+    def __init__(self, args, ctl):
+        QtWidgets.QLabel.__init__(self)
+        self.ctl = ctl
+        self.modify_argument()
+
+    def modify_argument(self):
+        self.ctl.set_argument_value("repo:ArgumentsDemo", "pyon_value", "123")
+        self.ctl.set_argument_value("repo:ArgumentsDemo", "number", 123e-6)
+        self.ctl.set_argument_value("repo:ArgumentsDemo", "integer", 123)
+        self.ctl.set_argument_value("repo:ArgumentsDemo", "string", "Hi")
+        self.ctl.set_argument_value("repo:ArgumentsDemo", "boolean", False)
+        self.ctl.set_argument_value("repo:ArgumentsDemo", "enum", "quux")
+        self.ctl.set_argument_value("repo:ArgumentsDemo", "scan", "ExplicitScan", 
```

Comment:
While I don't have a problem with this signature, it could be made more consistent with Scannable by making it a list in which the first object is the selected one, since that is how it is done there. This is debatable though.

Reply:
And in case only one is provided, a list would not be needed, again mirroring Scannable.

Reply:
This is a valid point. The format then will be set to something like this, 1st entry will be the default:
```
self.ctl.set_argument_value("repo:ArgumentsDemo", "scan",
                                    ["NoScan": {"value": 123, "repetitions": 2}, 
                                    "RangeScan" {"start": 200, "stop": 400, "npoints": 5, "randomize": True}]
```
@sbourdeauducq can you confirm?